packages/multimedia/ffmpeg/package.mk

```diff
@@ -35,6 +35,13 @@ PKG_BUILD_FLAGS="-gold"
 # Dependencies
 get_graphicdrivers
 
+if [ "$V4L2_SUPPORT" = "yes" ]; then
+  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET libdrm"
+  PKG_FFMPEG_V4L2="--enable-v4l2_m2m --enable-libdrm"
+else
+  PKG_FFMPEG_V4L2="--disable-v4l2_m2m --disable-libdrm"
```

Comment:
please remove `--disable-libdrm`, libdrm is needed by rkmpp decoder

Reply:
done